rosetta/rosetta/projects/vit/scripts/example_slurm_pretrain_pgle.sub

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+#SBATCH -A example
+#SBATCH -p partition
+#SBATCH -N 1                    # number of nodes
+#SBATCH -t 04:00:00             # wall time  (8 for backfill, 4 for Luna)
+#SBATCH -J jobname              # slurm job name
+#SBATCH --exclusive             # exclusive node access
+#SBATCH --mem=0                 # all mem avail
+#SBATCH --mail-type=FAIL        # only send email on failure
+#SBATCH --ntasks-per-node=8     # n tasks per machine (one task per gpu)
+#SBATCH --overcommit            # Needed for pytorch
```

Comment:
The comment says the line is 'Needed for pytorch'. Do we still need it here for JAX?